tuna/miopen_tables.py

```diff
@@ -518,6 +518,40 @@ class BNGolden(BASE, GoldenMixin):
   kernel_group = Column(Integer, nullable=True)
 
 
+class BenchmarkTable(BASE, GoldenMixin):
+  """benchmark table for framework and model parameters"""
+  __tablename__ = "benchmark_table"
+  __table_args__ = (UniqueConstraint("framework",
+                                     "model",
+                                     "batchsize",
+                                     "gpu_number",
+                                     "driver_cmd",
+                                     name="uq_idx"),)
+
+  framework = Column(String(length=128), nullable=False)
+  model = Column(String(length=128), nullable=False)
+  batchsize = Column(Integer, nullable=False, server_default="32")
+  gpu_number = Column(Integer, nullable=True, server_default="1")  #nullable?
+  driver_cmd = Column(String(length=128), nullable=False)
+
+
+class ConvBenchPerfTable(BASE, GoldenMixin):
+  """benchmark table for performance parameters"""
+  __tablename__ = "conv_benchmark_perf_table"
+  __table_args__ = (UniqueConstraint("solver",
+                                     "rocm_v",
+                                     "miopen_v",
+                                     name="uq_idx"),)
+
+  config = Column(Integer, ForeignKey("conv_config.id"), nullable=False)
+  benchmark = Column(Integer, ForeignKey("benchmark_table.id"), nullable=False)
+  kernel_time = Column(DOUBLE, nullable=False, server_default="-1")
+  solver = Column(String(length=128), nullable=True, server_default="")
```

Comment:
this needs to be an FKey to the solver table (similar to the find_table solver column)

Reply:
Should the onupdate/ondelete constraints remain there?

Reply:
yeah those are fine to stay